frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -1,30 +1,47 @@
 import * as React from 'react';
+import { Alert } from '@patternfly/react-core';
+import * as _ from 'lodash';
 import { Trans, useTranslation } from 'react-i18next';
+import { settleAllPromises } from '@console/dynamic-plugin-sdk/src/utils/promise';
 import { getActiveNamespace } from '@console/internal/actions/ui';
+import { Checkbox } from '@console/internal/components/checkbox';
 import {
   createModalLauncher,
   ModalTitle,
   ModalBody,
   ModalSubmitFooter,
 } from '@console/internal/components/factory/modal';
-import { history, resourceListPathFromModel } from '@console/internal/components/utils';
+import {
+  history,
+  ResourceLink,
+  resourceListPathFromModel,
+  StatusBox,
+} from '@console/internal/components/utils';
 import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
 import { useAccessReview } from '@console/internal/components/utils/rbac';
 import { ConsoleOperatorConfigModel } from '@console/internal/models';
 import {
   K8sKind,
+  K8sResourceCommon,
   K8sResourceKind,
+  modelFor,
+  referenceFor,
   k8sPatch,
   referenceForModel,
 } from '@console/internal/module/k8s';
 import { YellowExclamationTriangleIcon } from '@console/shared';
 import { CONSOLE_OPERATOR_CONFIG_NAME } from '@console/shared/src/constants';
 import { usePromiseHandler } from '@console/shared/src/hooks/promise-handler';
+import { useOperands } from '@console/shared/src/hooks/useOperands';
 import { getPatchForRemovingPlugins, isPluginEnabled } from '@console/shared/src/utils';
 import { GLOBAL_OPERATOR_NAMESPACE, OPERATOR_UNINSTALL_MESSAGE_ANNOTATION } from '../../const';
 import { ClusterServiceVersionModel, SubscriptionModel } from '../../models';
 import { ClusterServiceVersionKind, SubscriptionKind } from '../../types';
 import { getClusterServiceVersionPlugins } from '../../utils';
+import { OperandLink } from '../operand/operand-link';
+
+const anyOperandDeletionErrors = (operandDeletionErrors: OperandWithErrorProps[]): boolean =>
+  operandDeletionErrors.length > 0;
```

Comment:
Just inline this logic. This is only used in one place, is a very simple check, and is specific to a single component.